src/pages/setStandards/index.css

```diff
@@ -0,0 +1,35 @@
+.set-table .operatingRange {
```

Comment:
why there is an external css file? we can do only the necessary changes in index.css or follow the ANTD theme and configure.

Reply:
Needed to customize the table which is in figma design. Actually  I have added classname, so that it doesn't overwrite the default style on other tables. Please check once if there is anything to be modified.